geohisto/__main__.py

```diff
@@ -29,17 +36,29 @@ def main(at_date):
     compute_ancestors(towns)
     compute_populations(populations, towns)
     compute_parents(counties, towns)
+    if intercommunalities:
+        intercommunalities = load_intercommunalities(towns)
```

Comment:
Update the comment about the order if intercommunalities must be loaded (computed?) last/after towns.